docs/reference/vpc-association-policy.md

```diff
@@ -0,0 +1,85 @@
+# VpcAssociationPolicy API Reference
+
+## VpcAssociationPolicy
+
+VpcAssociationPolicy is a CRD that can be attached to a Gateway to define the ServiceNetworkVpcAssociation configuration.
+
+
+
+One of its option is `securityGroupIds`. it can control the inbound traffic from current cluster workloads to the gateway listeners. Please check the VPC lattice doc for more detail of this option. https://docs.aws.amazon.com/vpc-lattice/latest/ug/security-groups.html
+
+Recommended security group inbound rules:
```

Comment:
Is this an example or a general recommendation for all rules?

Reply:
general recommendation ? I referenced that one in the lattice doc

![image](https://github.com/aws/aws-application-networking-k8s/assets/32318664/2aa45935-1001-4b16-a91f-177080edfbd4)